yarn.lock

```diff
@@ -3215,6 +3312,11 @@
     consola "^2.15.0"
     node-fetch "^2.6.1"
 
+"@opentelemetry/api@^1.0.1":
```

Comment:
What is opentelemetry used for? Do we need it? Same goes for some of the dependencies below excluding the azure ones.